src/environment.rs

```diff
@@ -0,0 +1,103 @@
+/// Structure to deal with environment variables
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Environment {
+    /// Customized environment variables
+    pub vars: Vec<(String, String)>,
+    /// Define if the structure must inherit
+    pub inherit: bool,
+}
+
+impl Default for Environment {
+    fn default() -> Self {
+        Self {
+            vars: vec![],
+            inherit: false,
+        }
+    }
+}
+
+impl Environment {
+    /// Create a new Environment object with inherit
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// extern crate assert_cli;
+    ///
+    /// let e = assert_cli::Environment::inherit().compile();
+    /// let e_ = ::std::env::vars();
+    ///
+    /// assert_eq!(e, e_);
+    /// ```
+    pub fn inherit() -> Self {
+        Self {
+            vars: vec![],
+            inherit: true,
+        }
+    }
+
+    /// Create a new Environment object without inheriting
```

Comment:
Same about more context. Also, it can be confusing to describe something by what it is not.

So maybe `Create a new Environment independent of the current process's Environment`?